Developer.bundle

```diff
@@ -22,16 +22,12 @@ brew 'brew-cask-completion'
 brew 'bundler-completion'
 brew 'gem-completion'
 brew 'open-completion'
-brew 'vagrant-completion'
```

Comment:
I don't think we should remove this while we have `vagrant` installed below.

Reply:
@hkdobrev okay